src/main/java/de/tum/cit/ase/artemis_notification_plugin/CLIPlugin.java

```diff
@@ -15,7 +15,7 @@ public CLIPlugin() {
     }
 
     public static void main(String[] args) throws IOException {
-        CLIPlugin plugin = new CLIPlugin();
+        var plugin = new CLIPlugin();
```

Comment:
The `var` keyword was introduced with Java 10 (https://docs.oracle.com/javase/10/language/toc.htm).
As this plugin is currently based on version 8 there are compile errors.